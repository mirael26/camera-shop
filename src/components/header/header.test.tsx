import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import Header from './header';

jest.mock('./main-nav/main-nav', () => 'MainNav');
jest.mock('../search/search', () => 'Search');
jest.mock('./cart-button/cart-button', () => 'CartButton');

test('Header renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<Header/>);
  expect(asFragment()).toMatchSnapshot();
});
