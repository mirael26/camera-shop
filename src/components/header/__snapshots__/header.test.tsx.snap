// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<DocumentFragment>
  <header
    class="header"
    id="header"
  >
    <div
      class="container"
    >
      <a
        aria-label="Переход на главную"
        class="header__logo"
        href="/"
      >
        <svg
          aria-hidden="true"
          height="36"
          width="100"
        >
          <use
            xlink:href="#icon-logo"
          />
        </svg>
      </a>
      <div
        class="header__main-nav"
      >
        <mainnav />
      </div>
      <search />
      <cartbutton />
    </div>
  </header>
</DocumentFragment>
`;
